app/presenters/comment_presenter.rb

```diff
@@ -0,0 +1,46 @@
+class CommentPresenter < ApplicationPresenter
+  presents :comment
+  delegate :text, to: :comment
+
+  def submitters_name
```

Comment:
I'd probably just use the singular `submitter_name`. I'm assuming this was meant to be possessive, but that's not  idiomatic Ruby. Usually, variable / method names are singular when there's one instance and plural when there are multiple instances. Same with `submitters_link` below.

Reply:
👍